### PR TITLE
Added serverless support

### PR DESCRIPTION
**Pull request by cchostak**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -1,6 +0,0 @@
-[*]
```
Comment (victorfern91) [inline .editorconfig]: Why do you have deleted this file?

File: .env.local [hunk 2]
```diff
@@ -1,23 +1,23 @@
-APP_SERVER_PORT=3000
```
Comment (victorfern91) [inline .env.local]: doesn't change the indentation

File: api.apib [hunk 3]
```diff
@@ -3,7 +3,7 @@ HOST: https://trackcovid.pt/
 
 # trackcovid
 
-The API for the trackcovid app.
+The API for the tarckcovid app.
```
Comment (victorfern91) [inline api.apib]: remove that indentation

File: db/table.js [hunk 4]
```diff
@@ -53,14 +53,14 @@ module.exports = (table) => ({
 	 */
 	update: function (id, fields, client = null) {
 
-		const operation = Object.keys(fields).map((key, i) => `${key} = $${(i + 1)}`).join(', ')
-		const values = Object.values(fields).concat([id])
+		const operation = Object.keys(fields).map((key, i) => `${key} = ${(i + 1)}`).join(', ')
+		const values = Object.values(fields).push(id)
```
Comment (victorfern91) [inline db/table.js]: why you've changed this?

File: server.js [hunk 5]
```diff
@@ -22,6 +23,13 @@ app.use(session({
     saveUninitialized: true
 }));
 
+app.get('/', (req, res) => {
+    res.send({ application: 'trackovid', version: '1' });
+  });
+app.post('/getback', (req, res) => {
+    res.send({ ...req.body });
+});
+
```
Comment (victorfern91) [inline server.js]: why? this should be a new router

File: .editorconfig (hunk 1 above)
Comment (hugoduraes) [reply .editorconfig]: Yes, please don’t delete it as it’s useful for consistency across different editors

**Pull request closed without merging**